packages/jest-runner/src/index.ts

```diff
@@ -52,15 +60,17 @@ class TestRunner {
   ) {
     this._globalConfig = globalConfig;
     this._context = context || {};
+    this.eventEmitter = new Emittery();
```

Comment:
could we type this? https://github.com/sindresorhus/emittery/blob/master/readme.md#typescript

Reply:
Can type this once the `sendMessageToJest` typings are decided